cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -244,6 +244,9 @@ type Telemetry struct {
 type Logs struct {
```

Comment:
could this struct be replaced w/ `telemetry.LogsConfig`?

Reply:
Probably, it just depends on how much of `telemetry.LogsConfig` we want to take on. There are a lot of options in `telemetry.LogsConfig` and I believe we could switch to it later without a breaking change to end users.

Reply:
mostly im wondering if we could share the initialization code from that to avoid the duplication